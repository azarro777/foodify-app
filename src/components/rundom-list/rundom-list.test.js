import React from 'react';
import RundomList from './rundom-list';
import { shallow } from 'enzyme';

describe('Testing <RundomList/>', () => {
  it('RundomList have rendered correctly', () => {
    const appHeader = shallow(<RundomList />);
    expect(appHeader).toMatchSnapshot();
  });
});
