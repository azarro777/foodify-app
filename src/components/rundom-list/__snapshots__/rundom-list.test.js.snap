// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <RundomList/> RundomList have rendered correctly 1`] = `ShallowWrapper {}`;
